ocean_lib/web3_internal/event_listener.py

```diff
@@ -107,8 +111,13 @@ def _callback(event, *args):
 
     @staticmethod
     def watch_one_event(
-        event_filter, callback, timeout_callback, timeout, args, start_time=None
-    ):
+        event_filter: EventFilter,
+        callback: Callable,
+        timeout_callback: Callable,
```

Comment:
I think the `timeout_callback` should be `Optional[Callable]` based on the logic

```python
                    if timeout_callback is not None:
                        timeout_callback(*args)
```

Reply:
can be resolved.